react/components/FilterBar/FilterTag.js

```diff
@@ -1,7 +1,10 @@
 import React, { PureComponent, Fragment } from 'react'
 import PropTypes from 'prop-types'
 import merge from 'lodash/merge'
+import classNames from 'classnames'
 
+import useDeviceHook from '../utils/useDeviceHook'
```

Comment:
```suggestion
import useDevice from '../utils/useDevice'
```
The `use` word already means that it is a hook.